src/Features/CSharp/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -121,6 +121,13 @@ public override bool IsTriggerCharacter(SourceText text, int characterPosition,
 
         private INamedTypeSymbol GetCompletionListType(ITypeSymbol type, INamedTypeSymbol within, Compilation compilation)
         {
+            // PERF: None of the SpecialTypes include <completionlist> tags,
+            // so we don't even need to load the documentation.
```

Comment:
It could be worse--we only have to load documentation for the 1 type we're going to preselect and there typically isn't one. It would be pretty easy to have a dictionary of symbol to symbol, though.